src/app/style/styles.js

```diff
@@ -0,0 +1,67 @@
+export default {
```

Comment:
This module exports a piece of global mutable state, which is something
that we try hard to avoid. If you decide to keep this module, could you
consider instead exporting a function to create this sheet definition?